api2-war/src/main/resources/europeana.properties

```diff
@@ -26,6 +26,7 @@ mongodb.connectionUrl           = REMOVED
 mongodb.record.dbname           = REMOVED
 # to disable redirects, enter a non-existing database name
 mongodb.redirect.dbname         = REMOVED
+mongodb.max.connection.idle.time =
```

Comment:
Can we fill in a good default value here, that will probably work fine?

Reply:
yes, I will put the previous value as default. As it's tested already with stress tests